src/main/java/frc/trigon/robot/commands/CommandConstants.java

```diff
@@ -80,6 +96,21 @@ public static double calculateShiftModeValue(double minimumPower) {
         return 1 - squaredShiftModeValue * minimumShiftValueCoefficient;
     }
 
+    /**
+     * Calculates the target rotation value from the joystick's angle. Used for joystick oriented rotation.
+     * Joystick oriented rotation is when the robot rotates directly to the joystick's angle.
+     *
+     * @return the rotation value
+     */
+    private static MirrorableRotation2d calculateJoystickOrientedTargetAngle() {
+        final double joystickPower = 1 - Math.hypot(DRIVER_CONTROLLER.getRightX(), DRIVER_CONTROLLER.getRightY());
+        if (joystickPower > JOYSTICK_ORIENTED_ROTATION_DEADBAND)
+            return null;
```

Comment:
This doesn't make sense. If the joystick power is greater than the deadband it should move. This isn't the joystick power